src/status_im/ui/components/list_item/views.cljs

```diff
@@ -5,6 +5,7 @@
             [status-im.ui.components.list-item.styles :as styles]
             [status-im.utils.image :as utils.image]
             [status-im.ui.components.tooltip.views :as tooltip]
+            [status-im.ui.components.icons.vector-icons :as vector-icons]
```

Comment:
already imported under the alias "icons", feel free to change it to "vector-icons" though

Reply:
sure, i miss that :)